setup.py

```diff
@@ -36,8 +36,7 @@
     ],
     include_package_data=True,
     install_requires=[
-        'django',
-        'django-jsonview>=0.5.0',
+        'django>=2.2',
```

Comment:
Should we loosen this up to 
```suggestion
        'django>=2.0',
```